src/flow.jl

```diff
@@ -440,8 +440,10 @@ end
     q::Vector{T} | "m3 s-1"                 # river discharge (subgrid channel)
     q0::Vector{T} | "m3 s-1"                # river discharge (subgrid channel) at previous time step
     q_av::Vector{T} | "m3 s-1"              # average river discharge [m³ s⁻¹]
+    q_total_av::Vector{T} | "m3 s-1"        # average total (river and floodplain) discharge [m³ s⁻¹]
```

Comment:
While I agree that it is nice to have the option to keep the river discharge and combined (river+floodplain) discharge separate, I am thinking whether it is a good idea to have the combined output under a different variable name. If a user switches between using and not using 1d floodplains, this means that the output variables need to be changed as well. Is it therefore not more user-friendly to have the combined output under `q_av` and only the river discharge under something like `q_river_av`?

Regardless of the choice, it is good to have this explained in the documentation as well (it is currently missing from the parameter tables as well if I am right).

Reply:
Very good point, I agree and made the required changes. For only river discharge the parameter `q_channel_av` is used. I also added a bit more info (parameters and states) to the documentation.